tests/includes/utilities.inc

```diff
@@ -359,4 +359,36 @@ QUERY;
     return ($object instanceof FedoraObject);
   }
 
+  /**
+   * Asserts that the given PID represents a Fedora object.
+   *
+   * @param string $pid
+   *   The PID of the object.
+   *
+   * @return bool
+   *   TRUE if the object exists in the repository; FALSE otherwise.
+   */
+  public function assertObjectExists($pid) {
+    $object = islandora_object_load($pid);
+    $exists = $this->assertFedoraObject($pid);
```

Comment:
Should probably be passing `$object`, yeah? Also... Shouldn't be calling a static function non-statically... (use `static::assertFedoraObject($object)` kind of stuff, probably?